app/views/posts/show.html.erb

```diff
@@ -1,3 +1,20 @@
-<h1>Single Post</h1>
-<p><%= @placeholder_text %></p>
-<p>Find me in app/views/posts/show.html.erb</p>
+<article style="border: 1px solid gray">
+  <div style="display: flex; gap: 24px">
+    <h3>Post <%= @post.id %> by <%= @post.author.name %></h3>
+    <div style="display: flex; gap: 12px;">
+      <p>Number of Comments: <%= @post.comments_counter %></p>
+      <p>Number of Likes: <%= @post.likes_counter %></p>
+    </div>
+  </div>
+  <p><%= @post.text %></p>
+</article>
+
+<article style="border: 1px solid gray">
+  <% if @comments.present? %>
+    <% @comments.each do |comment| %>
+      <%= render partial: 'comments/comment', locals: { comment: comment } %>
+    <% end %>
+  <% else %>
```

Comment:
- [ ] Great job working on the views so far but I think it would be great if you made like the given wireframe since you should at least make the same layout so kindly adjust your element so you have a box containing the post content and above a no of comments and no of likes then a box to the comments. please make them the same given wireframe 🔥💪🏻🎉 I could see that you made the boxes but I think it would be great if you align the text for no of comments and no of likes in the right and so on to have in a better way✅💪🏻


![image](https://github.com/richie1988/blog-app/assets/58553711/ead20f39-8e2f-4e77-b06f-e530d1fe39b7)


- Your
![image](https://github.com/richie1988/blog-app/assets/58553711/998530e8-f28c-41d6-9a8a-450d94ae79c5)



- Given
![image](https://github.com/richie1988/blog-app/assets/58553711/1eb0c216-6315-43f6-893a-6dba41cf908d)